snuba/util.py

```diff
@@ -312,7 +220,7 @@ def tuplify(nested):
     return nested
 
 
-def conditions_expr(conditions, body, depth=0):
+def conditions_expr(dataset, conditions, body, depth=0):
     """
```

Comment:
Do you plan to eventually move this logic as well into the condition_expr method in the dataset class and remove the util one Or is this going to stay?

Reply:
I think this one will stay, there will be a common set of expresssions that we generate in the same way for every dataset, and then the other ones that we call out to the dataset-specific implementation for.